### ISSUE TITLE
Should have history of queries on local filesystem

### ISSUE THREAD

**Issue by evidanary** [tejal29/grepg#8]
We should log the query history in case someone wants to look at them on their local filesystem.

Perhaps the option should be called --log-queries-path=DIRECTORY_NAME
not generating a log when not supplied.

Filename could be something like grepg-queries.log with tab separated fields (IOS8901 date, hits, query) e.g. 
2016-04-01T13:01:02Z\truby open file\t3
2016-05-09T12:01:02Z\tpython open file\t0

**Comment by tejal29** [comment]
i am going to name it is as `--log-queries-dir`

**Comment by tejal29** [comment]
The format is going to be
**Date\tTopic\tQuery\tResults**
```
2016-04-01T13:01:02Z\truby\topen file\t3
2016-05-09T12:01:02Z\tpython\topen file\t0
```

**Comment by tejal29** [comment]
Fixed in 0.0.9.

Also added username in the log.
The format is going to be
Date\tUsername\tTopic\tQuery\tResults
```
2016-04-01T13:01:02Z\tevidanary\truby\topen file\t3
2016-05-09T12:01:02Z\ttejal29\tpython\topen file\t0
```
